README.md

```diff
@@ -80,7 +80,15 @@ import factory from 'trampss-redux-factory'
 export default factory()('id')('api.raw')('todos')
 ```
 
-Data will be stored into `state.api.raw.todos`
+Data will be stored into `state.api.raw.todos`.
+
+### Types
+Types are :
+  - `map` : your state is a hashmap, usefull to bind your API to Redux
+  - `uniq` : your state is an object, much simplier
+
+Default type is `map`.
```

Comment:
here, missing a line break (watch rendered markdown: it's in one line with the next one)